custom_dc/launch_cloud.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: Launch a custom site
+nav_order: 8
+parent:  Custom Data Commons
+---
+
+## Launch a custom site
+
+When you are ready to launch your site to external traffic, there are many tasks you will need to perform, including:
+
+-  Configure your Cloud Service to serve external traffic, over SSL. GCP offers many options for this; see [Mapping custom domains](https://cloud.google.com/run/docs/mapping-custom-domains).
+-  Optionally, restrict access to your service; see [Custom audiences (services)](https://cloud.google.com/run/docs/configuring/custom-audiences).
+-  Optionally, add a caching layer to improve performance. We have provided specific procedures to set up a Redis Memorystore in [Improve database performance](#redis).
+<!---  Optionally, add [Google Analytics](https://marketingplatform.google.com/about/analytics/) to track your website's usage. -->
+
+## Improve database performance {#redis}
+
+We recommend that you use a caching layer to improve the performance of your database. We recommend [Google Cloud Redis Memorystore](https://cloud.google.com/memorystore), a fully managed solution, which will boost the performance of both natural-language searches and regular database lookups in your site. Redis Memorystore runs as a standalone instance in a virtual private cloud (VPC) network, to which your Cloud Run service connects using a [VPC connector](https://cloud.google.com/vpc/docs/serverless-vpc-access).
+
+In the following procedures, we show you how to create a Redis instance in your project's "default" VPC network. You may wish to set up your own new VPC network; if so,  follow the documentation in [Create and manage VPC networks](https://cloud.google.com/vpc/docs/create-modify-vpc-networks) substitute that network name for `default` throughout these procedures.
```

Comment:
technically the redis instance doesn't live in the customer's VPC network, it lives in a Google-managed VPC, and connects to the customer's network ("default" or otherwise) via [direct peering](https://cloud.google.com/vpc/docs/vpc-peering)

Reply:
Might be sufficient to something like say: "we show you how to create a Redis instance **that connects to your project's** "default" VPC network."

Reply:
Thanks for this; I knew I didn't have it quite right. :-)  PTAL.